### ISSUE TITLE
Tasks tracking for extracting references from sources

### ISSUE THREAD

**Issue by lwjohnst86** [science-collective/scoping-review#32]
We'll use this issue to track the completion of extracting the specific sources (using R code). The final function should be to extract and save the data from the source, so we can include it in `_targets.R`.

- [x] #44 
- [x] #43 
- [x] #49
- [x] #45 
- [x] #42 
- [x] #46 
- [x] #47 
- [x] #41 
- [x] #48

**Comment by danielibsen** [comment]
I have run the search in pubmed, but there is an issue. Namely, that when I run the same search online on pubmed i get a lot more papers. Not sure what is going on, but will look into it.

**Comment by danielibsen** [comment]
I went through the documentation of all the packages for all the databases. However, for most there were different kinds of issues. Some needed usernames other were saying I did not have internet access and another ran the search but kept throwing an error because of some unexpected symbol in the papers it identified. Most of them also use their own way for formulating the searches and do not allow for exactly the same search options. Anyway, there is certainly a lot to learn here.
